src/Model/Tag.php

```diff
@@ -432,53 +437,53 @@ private static function getTypeForHash(string $hash)
 	/**
 	 * Create implicit mentions for a given post
 	 *
-	 * @param integer $uri_id
-	 * @param integer $parent_uri_id
+	 * @param integer $uriId URI Id
+	 * @param integer $parentUriId Parent URI id
+	 * @return void
 	 */
-	public static function createImplicitMentions(int $uri_id, int $parent_uri_id)
+	public static function createImplicitMentions(int $uriId, int $parentUriId)
 	{
 		// Always mention the direct parent author
-		$parent = Post::selectFirst(['author-link', 'author-name'], ['uri-id' => $parent_uri_id]);
-		self::store($uri_id, self::IMPLICIT_MENTION, $parent['author-name'], $parent['author-link']);
+		$parent = Post::selectFirst(['author-link', 'author-name'], ['uri-id' => $parentUriId]);
+		self::store($uriId, self::IMPLICIT_MENTION, $parent['author-name'], $parent['author-link']);
 
 		if (DI::config()->get('system', 'disable_implicit_mentions')) {
 			return;
 		}
 
-		$tags = DBA::select('tag-view', ['name', 'url'], ['uri-id' => $parent_uri_id, 'type' => [self::MENTION, self::EXCLUSIVE_MENTION, self::IMPLICIT_MENTION]]);
+		$tags = DBA::select('tag-view', ['name', 'url'], ['uri-id' => $parentUriId, 'type' => [self::MENTION, self::EXCLUSIVE_MENTION, self::IMPLICIT_MENTION]]);
 		while ($tag = DBA::fetch($tags)) {
-			self::store($uri_id, self::IMPLICIT_MENTION, $tag['name'], $tag['url']);
+			self::store($uriId, self::IMPLICIT_MENTION, $tag['name'], $tag['url']);
 		}
 		DBA::close($tags);
 	}
 
 	/**
 	 * Retrieves the terms from the provided type(s) associated with the provided item ID.
 	 *
-	 * @param int       $item_id
-	 * @param int|array $type
-	 * @return array
+	 * @param int       $uriId URI id
```

Comment:
You didn't have to do this.
```suggestion
	 * @param int   $uriId
```